examples/typescript-interop/.gitignore

```diff
@@ -0,0 +1,9 @@
+build
+node_modules
+hello.js
+.vscode
+libhello
+*.dylib
+*.so
+*.dll
+*.o
```

Comment:
Should most of these just be added at the root level of the repo and ignored everywhere?

Reply:
Good idea! Moved some in 71331cc9453e8d027601c56f425881376a8c125f